src/framework/components/camera/component.js

```diff
@@ -148,11 +148,11 @@ class CameraComponent extends Component {
      * const shaderPassId = camera.setShaderPass('custom_rendering');
      *
      * material.onUpdateShader = function (options) {
-     *    if (options.pass === shaderPassId) {
-     *        options.litOptions.normalMapEnabled = false;
-     *        options.litOptions.useSpecular = false;
-     *    }
-     *    return options;
+     * if (options.pass === shaderPassId) {
+     * options.litOptions.normalMapEnabled = false;
+     * options.litOptions.useSpecular = false;
+     * }
```

Comment:
is this required? It does not look as good as before.

Reply:
I think this is a bug in eslint_plugin_jsdoc. I'll investigate.

Reply:
this has been fixed / avoided in https://github.com/playcanvas/engine/pull/5472